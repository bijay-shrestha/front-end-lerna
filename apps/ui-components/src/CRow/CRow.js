import React from 'react';
import Row from "react-bootstrap/Row";
import PropTypes from 'prop-types';

const CRow = props => {
    const {
        as,
        bsPrefix,
        children,
        id,
        noGutters
    } = props;
    return (
        <Row
            as={as}
            bsPrefix={bsPrefix}
            children={children}
            id={"row_".concat(id)}
            noGutters={noGutters}
        />
    )
};

CRow.propTypes = {
    as: PropTypes.elementType,
    bsPrefix: PropTypes.string,
    children: PropTypes.node,
    id: PropTypes.string.isRequired,
    noGutters: PropTypes.bool
};

export default CRow;