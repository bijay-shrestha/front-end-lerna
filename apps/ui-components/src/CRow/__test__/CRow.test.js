import CRow from "../CRow";

describe('CRow Component Tests', () => {

    let wrapper;
    beforeEach(() => {
        wrapper = shallow(<CRow/>);
    });

    test('if CRow Component is defined', () => {
        expect(wrapper.find('CRow')).toBeDefined();
    });

});