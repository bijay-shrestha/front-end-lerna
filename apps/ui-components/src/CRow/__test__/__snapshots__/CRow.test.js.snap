// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CRow Component Tests Snapshot Testing should match snapshot 1`] = `
<FormRow
  id="row_test"
/>
`;
